### PR TITLE
Add listfav and clearfav

### LINKED ISSUE #101: /listfav and /clearfav features for FavList
list out favorites with index.
clear all favorites.

### PR DESCRIPTION
**Pull request by mrwsy1**
/listfav
/clearfav

### CODE REVIEW

**Review by EthanWong2212 — APPROVED** [review]

File: src/main/java/seedu/duke/logic/commands/favcommand/ClearFavCommand.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+import seedu.duke.favorite.FavList;
+import seedu.duke.logic.commands.Command;
+
+public class ClearFavCommand extends Command {
+
+    @Override
+    public void executeCommand() {
+        FavList.clearFav();
+    }
```
Comment (wamikamalik) [inline src/main/java/seedu/duke/logic/commands/favcommand/ClearFavCommand.java:11]: Would it be better to call the Ui method to print the message here instead?

Comment (mrwsy1) [reply src/main/java/seedu/duke/logic/commands/favcommand/ClearFavCommand.java:11]: Can, but I was following the structure from ethan's skeleton code, the methods are all in FavList

Comment (wamikamalik) [reply src/main/java/seedu/duke/logic/commands/favcommand/ClearFavCommand.java:11]: I see, okie!

**Review by wamikamalik — APPROVED** [review]
LGTM!

**Review by EthanWong2212 — APPROVED** [review]
looks good to merge

**Pull request merged**